gulpfile.js

```diff
@@ -27,7 +27,8 @@ gulp.task('mocha', function () {
   return gulp
     .src(['./test/**/*', '!./test/temp'])
     .pipe(mocha({
-      reporter: 'spec'
+      reporter: 'spec',
+      timeout: Infinity
```

Comment:
@ilanbiala Actually, right now, as `timeout(0)` has been removed from the tests, if you run `app.test.js` with mocha, it will 80% fail.

So, I'd opt for putting `this.timeout(0)` back into each of the tests :)